ti/instrumentation/instrumentation.go

```diff
@@ -219,3 +220,15 @@ func GetCmd(ctx context.Context, config *api.RunTestConfig, stepID, workspace st
 	command := fmt.Sprintf("%s\n%s\n%s", config.PreCommand, testCmd, config.PostCommand)
 	return command, nil
 }
+
+// InjectReportInformation add default test paths information to ruby and python when test runner is invoked without a value
+// This serves as a default
+func InjectReportInformation(r *api.StartStepRequest) {
+	switch strings.ToLower(r.RunTest.Language) {
+	case "ruby", "python":
+		if r.RunTest.Args == "" && len(r.TestReport.Junit.Paths) == 0 {
+			r.TestReport.Junit.Paths = []string{fmt.Sprintf("**/%s*", common.HarnessDefaultReportPath)}
```

Comment:
How would you handle RunTests with parallelism? Are they writing to a separate folder or separate files?

The above regex won't handle `harness_test_results_0.xml` and your runner is not handling writing to a separate directory for each step.

Reply:
for python, it will append to the end (harness_test_results.xml0) in the agent. Ruby still TBD. 

This should only matter for step level. I can add the concurrent index later.

Reply:
Shouldn't it be `harness_test_results_0.xml`

Reply:
index at the end will be easier to handle. We are suggesting the same to the customer for the parallelism. The report does not need to end with xml